tests/sampleTests/replaySample.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { expect, test } from '@playwright/test';
+import { Polly } from '@pollyjs/core';
+
+import { submitPaymentResult } from '../api.mock';
+import { replayInitializer } from '../polly.global.setup';
+
+// Uncommmnet to continue in a headed browser
+// test.use({
+//     headless: false,
+//     viewport: { width: 1000, height: 1000 },
+// });
+
+test.describe('Replay mode sample', () => {
+
+    let polly: Polly;
+
+    test.beforeEach(async ({ page }) => {
+        // 1. Setup PollyJS
+        polly = await replayInitializer({
+            playwrightContext: page,
+            recordingName: 'sample',
+            storeURL: 'https://my-dev-store-745516528.store.bcdev',
+        });
+
+        await page.goto('http://localhost:8080/');
+    });
+
+    test('Bigpay Test Payment Provider is working', async ({page}) => {
+
+        // 1. Setup PollyJS
+        // Since we put it into test.beforeEach(), don't have to initialize again
+        // const polly = await replayInitializer({
+        //     playwrightContext: page,
+        //     recordingName: 'test-recording',
+        //     storeURL: 'https://my-dev-store-745516528.store.bcdev',
+        // });
+
+        // 2. Serve additional static files with Playwright
+        await page.route('**/checkout/payment/hosted-field?**', route => route.fulfill( {status: 200, path: './tests/sampleTests/_support/hostedField.html' } ));
```

Comment:
Also, what about HTML responses, could this be recorded as well?

Reply:
AFAIK, PollyJS only works with Fetch/XHR, therefore, an automatically managed HAR is around 200kb in our case.

In addition, I changed the `parentOrigin` attribute in hostedField.html to make iframe communication valid.